funsor/distribution.py

```diff
@@ -335,24 +337,30 @@ def __call__(self, cls, args, kwargs):
             ast_fields = cls._funsor_ast_fields
         except AttributeError:
             ast_fields = cls._funsor_ast_fields = getargspec(cls.__init__)[0][1:]
+        kwargs = {name: value for pairs in (zip(ast_fields, args), kwargs.items())
+                  for name, value in pairs}
```

Comment:
This resolves an issue where BinomialProbs/MultinomialProbs takes `probs` as its first argument in NumPyro (though `Binomial`, `Multinomial`'s arguments still have the same order as in Pyro).